src/main/java/com/dev/cinema/model/dto/MovieSessionRequestDto.java

```diff
@@ -1,8 +1,14 @@
 package com.dev.cinema.model.dto;
 
+import javax.validation.constraints.Min;
```

Comment:
wrappers are not supported by this annotation

Reply:
fixed